documentation/specifications/README.md

```diff
@@ -1,29 +1,29 @@
 # Protocol Specification
 
-This document describes the protocol specifications for the Diem Payment Network (LPN). The intended audience for this document are as follows:
+This document describes the protocol specifications for the Aptos Payment Network (LPN). The intended audience for this document are as follows:
```

Comment:
so weird Aptos PN and LPN ... Aptos blockchain

Reply:
Ughh.. yeah, there's gonna be a bunch of these stragglers. I can try and do some grepping today to find missed patters and/or references to Libra :P